src/data-import.ts

```diff
@@ -0,0 +1,331 @@
+/**
+ * DataImportService
+ * Handles inventory data import functionality for cross-device synchronization
+ */
+
+import type { InventoryItems } from './inventory.js';
+import type { ExportData } from './data-export.js';
+
+export interface ImportData extends ExportData {
+  // Import data has the same structure as export data
+}
+
+export interface ImportOptions {
+  mergeStrategy: MergeStrategy;
+  validateSchema: boolean;
+  createBackup: boolean;
+}
```

Comment:
Remove this interface and:

1. Assume that, as reported [here](https://github.com/gabrieledarrigo/mtn-warehouse/pull/21/files#r2249367503), the strategy is always "replace"
2. Always validate the schema
3. Never create a backup